apps/ledger-live-desktop/src/renderer/screens/settings/sections/Accounts/HiddenNFTCollections/index.tsx

```diff
@@ -109,8 +109,8 @@ export default function HiddenNftCollections() {
           {visibleCollections.map(collectionId => {
             const { accountId, contractAddress } = decodeCollectionId(collectionId);
             const network = (Object.keys(collections).find(
```

Comment:
Here the type assertions could be removed by defining an alternative to `Object.keys` with a decent type definition:
```ts
const keys: <T extends object>(obj: T) => (keyof T)[] = Object.keys;
```

Then:
```ts
const network: SupportedBlockchains =
  keys(collections).find(key => collections[key][collectionId]) ?? Blockchain.Ethereum;
```